test/primitives/array-element-fantasy-land.js

```diff
@@ -0,0 +1,56 @@
+var expect = require('../spec-helper').expect;
+var minim = require('../../lib/minim').namespace();
+var R = require('ramda');
+
+var ArrayElement = minim.getElementClass('array');
+
+describe('ArrayElement', function() {
```

Comment:
For tests I recommend using Fantasy-Land laws: https://github.com/fantasyland/fantasy-laws

Here is an example how to do it. These laws grantee that your implementation implements various mathematical properties like e.g. reflexivity, transitivity, symmetry

Reply:
I like the idea, but I am struggling to get this done. There's a lot to grasp in fantasy-laws, along with its dependencies, jsverify and sanctuary.

I am wondering if perhaps we can proceed without fantasy-laws to get this out sooner and not block other work and perhaps we can sync up while I'm in Prague in few weeks @char0n. We could walk through implementing one of the tests together to help me understand how to fit it all together?

Reply:
I've already improved the tests and moved away from Ramda to calling the fantasy land methods directly via `fantasy-land` package to catch typos in the method names.

Reply:
Yep as discussed this is just fine. We'll sync on laws personally if that is something you want to pursue.